### PR TITLE
Allow AVPlayer to play even when user has device in silent mode

### LINKED ISSUE #1358: [BUG] MediaElement does not play audio if 'Silent mode' is switched on in iOS
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

iOS currently does not play audio even when a user has requested to play audio / video

### Expected Behavior

Audio is expected to be played, despite silent mode having been switched on.

### Steps To Reproduce

1. Switch silent mode on for your iOS device / Simulator
2. Launch the official CommunityToolkit sample app and go to the MediaElement view
3. Video should play but Audio should be muted (Even if the volume slider in iOS / MediaElement is at 100%)

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui

### Environment

```markdown
- .NET MAUI CommunityToolkit (MediaElement): 2.0.0
- OS: Windows 11
- .NET MAUI: 7.0.92
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ArchangelWTF**
### Description of Change ###

Modifies MediaElement's MediaManager on iOS to allow iOS devices to play audio in 'Silent Mode'.

 ### Linked Issues ###

 - Fixes #1358

 ### PR Checklist ###
 
 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [x] Has tests (if omitted, state reason in description)
 - [x] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [x] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

This implementation was tested on a physical iOS device and works fine.

### CODE REVIEW

**Comment by ArchangelWTF** [comment]
@dotnet-policy-service agree

File: src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs [hunk 1]
```diff
@@ -63,6 +63,11 @@ public interface IMediaElement : IView
 	/// </summary>
 	bool ShouldMute { get; set; }
 
+	/// <summary>
+	/// Gets or sets if the audio should bypass silent mode if a user has it enabled.
+	/// </summary>
+	bool ShouldBypassSilentMode { get; set; }
```
Comment (pictos) [inline src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs:69]: I saw just an implementation for apple platform, how about Windows and Android?

**Review by pictos — CHANGES_REQUESTED** [review]

File: src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs (hunk 1 above)
Comment (brminnick) [reply src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs:69]: We're going to discuss this in today's standup! I'll circle back with the resolution later today.

**Comment by brminnick** [comment]
Hi @ArchangelWTF! We had a good long discussion on this in [today's standup](https://www.youtube.com/watch?v=6SmUzY94LZA). 

We've decided that it'll be best to showcase your code in the [official MediaElement docs](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/mediaelement).

I'll open up the Docs PR now and tag you in it for review.

**Comment by brminnick** [comment]
Closing in favor of adding the workaround to the Official Docs: https://github.com/brminnick/CommunityToolkitDocs/pull/3

**Pull request closed without merging**